pkg/webhook/validation/uservalidation.go

```diff
@@ -137,6 +140,49 @@ func isMemberClusterUpdated(currentMC, oldMC fleetv1alpha1.MemberCluster) (bool,
 	return currentMCHash != oldMCHash, nil
 }
 
+func ValidateCRP(crp fleetv1alpha1.ClusterResourcePlacement) error {
+	allErr := make([]error, 0)
```

Comment:
There is already CRP validator https://github.com/Azure/fleet/blob/main/pkg/utils/validator/clusterresourceplacement.go

Reply:
Need test too

Reply:
Using validator.ValidateClusterResourcePlacementAlpha and added UTs